### ISSUE TITLE
Steam Deck Support

### ISSUE THREAD

**Issue by Morpheus0x** [networkMe/missinghud2#96]
I got my Steam Deck today and started playing TBOI as my first game. I haven't played in a while and wanted to setup the 2 most useful mods: External Item Descriptions and this one. The first is not a problem since it is in the steam workshop. However Missing HUD 2 isn't and has to be injected. Is injecting a DLL the only way to get this mod to work? Something like replacing a DLL in the game files would be a usable solution for now, but isn't perfect since every update would overwrite it. I am happy to test the mod on my Steam Deck.

TL;DR
Please make this mod compatible with the Steam Deck.
